tests/node/node.go

```diff
@@ -24,11 +24,11 @@ import (
 	"os/exec"
 	"strings"
 
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
+	test "github.com/onsi/ginkgo"
+	matcher "github.com/onsi/gomega"
 )
 
-// TODO: Use job with a node affinity instead
+// RunOnNode Use job with a node affinity instead
```

Comment:
I think we should keep the TODO, and add an additional function description

```
// RunOnNode performs bash shell command on node
// TODO: Use job with a node affinity instead
```

Reply:
@RamLavi Thanks! I've rebased the PR, Let's see the CI run now.